### DISCUSSION TITLE
espixelstick url

### DISCUSSION THREAD

**Discussion by NIcozozo57** [forkineye/ESPixelStick#418] [Q&A] [answered]
Hi, 
I wonder if there is a way to initiate a reboot of the pixelstick with a GET command like ; http://192.168.0.20/Reboot 
or any other way.

I'm asking that because, my Wemos D1 mini work perfeclty in unicast (with some lag) but in unicast, sometime they are unresponsive on multicast, and a reboot fix the problem.

Regards

**Comment by forkineye** [comment, marked as answer]
Currently, there is no exposed API.  Everything between the UI is done via websockets.
